doc/examples/pilz_industrial_motion_planner/pilz_industrial_motion_planner.rst

```diff
@@ -265,39 +263,84 @@ Planning results in ``moveit_msg::MotionPlanResponse``
 -  ``group_name``: name of the planning group
 -  ``error_code/val``: error code of the motion planning
 
-Example
--------
+Examples
+--------
 
 By running
 
 ::
 
-    roslaunch prbt_moveit_config demo.launch
+    ros2 launch moveit2_tutorials demo.launch.py rviz_config:=panda_hello_moveit.rviz
 
-the user can interact with the planner through rviz.
+you can interact with the planner through the RViz MotionPlanning panel.
 
 .. figure:: rviz_planner.png
    :alt: rviz figure
 
+To use the planner through the MoveGroup Interface, refer to
+:ref:`the MoveGroup Interface C++ example </doc/examples/pilz_industrial_motion_planner/src/pilz_move_group.cpp>`.
```

Comment:
These are failing the htmlproofer check. Is there a good way to link to a C++ source file within the repo without just using the GitHub link as is?

Reply:
Have you tried using `codedir` from in [conf.py](https://github.com/ros-planning/moveit2_tutorials/blob/main/conf.py) to help with the link?

For example, this links to a launch file within the MoveIt Tutorials repo:
```
:codedir:`here<tutorials/pick_and_place_with_moveit_task_constructor/launch/mtc_demo.launch.py>`.
```

See [here](https://github.com/ros-planning/moveit2_tutorials/blame/main/doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.rst#L517) to see that line in context. I don't think I can permalink directly to an RST file, so I've just highlighted the line in the blame.

See [here](https://moveit.picknik.ai/humble/doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.html#launch-files) for what it looks like in the tutorial.

Reply:
That's perfect -- thank you so much!